modules/oe_content_entity/modules/oe_content_entity_organisation/modules/oe_content_organisation_reference/oe_content_organisation_reference.info.yml

```diff
@@ -0,0 +1,19 @@
+name: OpenEuropa Organisation Reference
+description: Allows to reference organisations in very specific cases, such as as a contact information.
+package: OpenEuropa
+type: module
+core: 8.x
+
+dependencies:
+  - oe_content:oe_content
+  - oe_content:oe_content_entity
+  - oe_content:oe_content_entity_contact
+  - oe_content:oe_content_organisation
```

Comment:
Add here 
```
inline_entity_form
path
```
As form display requires them.

Reply:
I can't see that the form here would require this.
![image](https://user-images.githubusercontent.com/22004498/98666276-13687980-234d-11eb-94d7-3a8d6107e6b0.png)

Reply:
Right, my bad.